apps/video/js/video.js

```diff
@@ -116,6 +116,7 @@ document.addEventListener('visibilitychange', function visibilityChange() {
   else {
     if (playerShowing) {
       setControlsVisibility(true);
+      setVideoAccessibility(false);
```

Comment:
Since 'setVideoAccessibility' is being called everywhere where 'setControlsVisibility' is being called, I suggest moving the 'setVideoAccessibility' call into 'setControlsVisibility'